cmd/jag/commands/container.go

```diff
@@ -140,6 +160,8 @@ func ContainerInstallCmd() *cobra.Command {
 	cmd.Flags().StringArrayP("define", "D", nil, "define settings to control container on device")
 	cmd.Flags().String("assets", "", "attach assets to the container")
 	cmd.Flags().IntP("optimization-level", "O", -1, "optimization level")
+	cmd.Flags().String("interval", "", "check interval for container starts (e.g., '30s', '5m', '1h'). "+
+		"When specified, Jaguar will start the container at the specified interval if it has previously exited.")
```

Comment:
I think the interval help is too complicated to fit nicely in the short help.

I would add something like the following to the big help above:
```
If an interval is given, the container is started at the given frequency. If the container is
still running, then all subsequent starts are simply delayed. If a container should run
continuously, an interval of `1s` is a good choice. If the container runs more than 1 second
it will be restarted automatically, but if it crashes immediately, then it won't be restarted
for 1 second, giving the system more room to recover.
```

```suggestion
	cmd.Flags().String("interval", "", "interval for container starts (like '30s', '5m', or '1h').")
```

Reply:
Done in https://github.com/toitlang/jaguar/pull/624/commits/9f21871beeffd0bfacf76ab3820f1ab1d6a530f0